SR2021/2020-12-10-team-progress-checklist.md

```diff
@@ -0,0 +1,21 @@
+---
+to: Team Leaders
+subject: Student Robotics Team Progress Checklist
+---
+Hello there!
+
+There are only 5 weeks until our [first round of league matches](https://studentrobotics.org/events/sr2021/league-1/) so we wanted to check in and make sure you're making good progress! We've put together the below checklist of things that should have happened by now to help you figure out if you're on track for success.
+
+ - Team Leader should be in Discord
+ - Team should be in Discord
+ - (If multiple teams) Competitors have formed separate teams and have the correct team role in Discord
+ - Microgames from Kickstart completed
+ - You have a robot that moves
+ - Everyone in the team has access to the team's code
+ - Everyone in the team has run the code
```

Comment:
I think we should probably expand these into slightly more full sentences.

```suggestion
 - Team Leaders are be in Discord, and have the "Team Leader" role
 - Competitors are be in Discord
 - (If multiple teams) Competitors have formed separate teams and have the correct team role in Discord
 - Microgames from Kickstart are completed
 - You have a robot that moves, however intelligently
 - Everyone in the team has access to the team's code and can collaborate
 - Everyone in the team has run the code and understands how it works
```

Reply:
Regarding the last of these -- measuring understanding is potentially fairly hard. I think there's value in the team members all having run the code even if they don't yet understand it. Perhaps break that into two parts?